src/Network/DO.hs

```diff
@@ -22,41 +25,57 @@ module Network.DO(
 import           Control.Exception            (catch, throw)
 import           Control.Monad.Trans.Free
 import           Data.IP
-import qualified Network.DO.Commands          as C
-import qualified Network.DO.Domain            as C
-import qualified Network.DO.Droplets.Commands as C
 import           Network.DO.Droplets.Utils
-import qualified Network.DO.IP.Commands       as C
 import           Network.DO.Names
 import           Network.DO.Net
 import           Network.DO.Pairing
 import           Network.DO.Pretty
-import           Network.DO.Types
+import           Network.DO.Types             hiding(name, tagResources)
 import           Network.REST
 import           System.Environment           (getEnv)
 import           System.IO.Error              (isDoesNotExistError)
+import qualified Network.DO.Commands          as C
+import qualified Network.DO.Domain            as C
+import qualified Network.DO.Droplets.Commands as C
+import qualified Network.DO.IP.Commands       as C
+import qualified Network.DO.Tags.Commands     as C
 
-type Command w a = FreeT (C.DO :+: C.DropletCommands :+: C.IPCommands :+: C.DomainCommands) (RESTT w) a
+type Command w a = FreeT (C.DO :+: C.DropletCommands :+: C.IPCommands :+: C.DomainCommands :+: C.TagsCommands) (RESTT w) a
 
+--
+-- KEYS
+--
 listKeys :: (Monad w) => Command w [Key]
 listKeys = injl C.listKeys
 
+--
```

Comment:
If you want to add comments there, it might make more sense to haddock comments that will be processed and could serve to better organise the file. This requires adding  `-- |` before the comment

Otherwise, I am not sure the comments are really useful as is...